### PR TITLE
Added dynamic news from newsroom.eclipse.org

### PR DESCRIPTION
**Pull request by autumnfound**
Added jquery plugin as a dependency. This can be removed once it is
added to the base theme. Added a call to inject news on news list
container, pulling announcements and community news.

Signed-off-by: Martin Lowe <martin.lowe@eclipse-foundation.org>

### CODE REVIEW

**Comment by autumnfound** [comment]
Question about the View All link. Right now I put in the best guess for what the URL would be, but I don't think we have a filtered view of the news yet by site publish target. If there is a proper link could you leave an example in the comments of this PR?

**Comment by chrisguindon** [comment]
> Question about the View All link. Right now I put in the best guess for what the URL would be, but I don't think we have a filtered view of the news yet by site publish target. If there is a proper link could you leave an example in the comments of this PR?

The url should be https://ecdtools.eclipse.org/news/. We will need to create a layout page which will implement the news widget with pagination.

The same should be done for events https://ecdtools.eclipse.org/events/.

I expect us to remove all news/event markdown file when submit a PR for installing the news/event widget.

File: webpack.mix.js [hunk 1]
```diff
@@ -25,6 +25,8 @@ mix.scripts([
     './node_modules/jquery-match-height/dist/jquery.matchHeight-min.js',
     './node_modules/feather-icons/dist/feather.min.js',
     './node_modules/cookieconsent/src/cookieconsent.js',
+    './node_modules/jquery-eclipsefdn-api/dist/jquery.eclipsefdn-api.min.js',
+    './node_modules/jquery-eclipsefdn-api/dist/jquery.eclipsefdn-igc.min.js',
```
Comment (chrisguindon) [inline webpack.mix.js:29]: Let's remove this for now.

File: templates/event-list-format.mustache [hunk 2]
```diff
@@ -0,0 +1,11 @@
+<div class="row news-item text-center padding-bottom-15">
+{{#events}}
```
Comment (chrisguindon) [inline templates/event-list-format.mustache:2]: How about we store our templates in js/templates. or js/tpl ?

File: package.json [hunk 3]
```diff
@@ -25,6 +25,7 @@
     "build_netlify": "NODE_ENV=production npm run disallow_robots_txt && npm run production && npm run fetch_blogs && npm run fetch_projects"
   },
   "dependencies": {
-    "eclipsefdn-hugo-solstice-theme": "0.0.58"
+    "eclipsefdn-hugo-solstice-theme": "0.0.63",
+    "jquery-eclipsefdn-api": "0.0.29"
```
Comment (chrisguindon) [inline package.json:29]: I will create a release of hugo to include "jquery-eclipsefdn-api": "0.0.29". Let's remove it!

**Pull request merged**